app/dashboard/templates/quickstart.html

```diff
@@ -59,11 +59,7 @@ <h4 class="title">{% trans "Quickstart Guide" %}</h4>
             <li>{% trans "4) Confirm the transaction in metamask" %}</li>
           </ol>
           <p>
-            {% blocktrans %}
-            Once confirmed on blockchain, your bounty is posted to Gitcoin!
-            Wait for developers across the world to discover your bounty and
-            start working on your issue.
-            {% endblocktrans %}
+            {% trans "Once confirmed on blockchain, your bounty is posted to Gitcoin! Wait for developers across the world to discover your bounty and start working on your issue." %}
```

Comment:
@SaptakS Wanna break this up into multiple lines? Avoids horizontal scrolling 😅

Reply:
I had this broken but then it was causing problem with `{% trans`. Instead of rendering, it was showing `{% trans` as normal text.